app/package-lock.json

```diff
@@ -1,87 +1,46 @@
 {
   "name": "app",
   "version": "0.1.0",
-  "lockfileVersion": 3,
+  "lockfileVersion": 1,
```

Comment:
It seems your npm is not up-to-date. Are you using the devcontainer or your local Node environment?

Reply:
Nope :eyes: sorry haha, will do